client/src/views/Game.vue

```
@@ -72,10 +138,10 @@ async function drawCard() {
 }
 
 async function playCard(cardId:number) {
-  console.log(player.playerGameId+" -> "+ game.value?.currentRound?.currentPlayer)
-  if (player.playerGameId === game.value?.currentRound?.currentPlayer) {
+  console.log(myPlayerId+" -> "+ game.value?.currentRound?.currentPlayer)
+  if (myPlayerId.value === game.value?.currentRound?.currentPlayer) {
     let color = undefined;
-    if (game.value?.currentRound?.players[player.playerGameId - 1].hand.cards[cardId].type === Type.Wild) {
+    if (game.value?.currentRound?.players[(myPlayerId.value ?? 1) - 1].hand.cards[cardId].type === Type.Wild) {
```

?? 1 => can cause silent errors, which we will look forever